src/constants/default-settings.js

```diff
@@ -42,6 +42,7 @@ import {
   Cancel
 } from 'components/common/icons';
 import {getHTMLMapModeTileUrl} from 'utils/utils';
+import { emptyLocales } from '../localization/locales';
```

Comment:
```suggestion

```

Reply:
We wont use this anymore

Reply:
@leodavinci1 if you are not using it, please remote it